lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -27,7 +27,7 @@ def run!(args)
     end
 
     def analyze
-      @matches = @output.chomp.split("\n\n").map { |m| m.split("\n") }
+      @matches = @output.chomp.split("\n\n").map { |m| m.gsub(/\n\s+\[/, "SPLIT_ME_HERE_PLEASE[").split("SPLIT_ME_HERE_PLEASE") }
```

Comment:
Yes, splitting on the new line does not work with an embedded new line. With an embedded new line in the message, you would end up with two matches (and therefore two smells).  This fix is to address that.